MaxText/layers/moe.py

```diff
@@ -639,7 +639,13 @@ def gmm(inputs, kernel, group_sizes, expert_assignments):
         quant_dg = self.quant.quant_dg
         lhs_quantize_dtype = quant_dg.fwd.dg_quantizer.lhs.numerics.get_dtype()
         rhs_quantize_dtype = quant_dg.fwd.dg_quantizer.rhs.numerics.get_dtype()
-
+      if self.config.use_qwix_quantization:
+        if self.config.quantization == "int8":
+          lhs_quantize_dtype = jnp.int8
+          rhs_quantize_dtype = jnp.int8
+        elif self.config.quantization == "fp8":
+          lhs_quantize_dtype = jnp.float8_e4m3fn
+          rhs_quantize_dtype = jnp.float8_e4m3fn
```

Comment:
Shall we have assertion for this `quantization` in pyconfig.py? otherwise, it could be silently out.